playbooks/robusta_playbooks/autoscaler.py

```diff
@@ -50,7 +50,7 @@ def alert_on_hpa_reached_limit(
     event: HorizontalPodAutoscalerChangeEvent, action_params: HPALimitParams
 ):
     """
-    Notify when the HPA reaches it's maximum replicas and allow fixing it.
+    Notify when the HPA reaches its maximum replicas and allow fixing it.
```

Comment:
`it's` = `it is`, this is a grammar fix and is entirely manual (the spell checker at this time only looks at 3+ letter words excluding trailing apostrophe+a letter).